### PR TITLE
Feature/add toggle which changes the language

### PR DESCRIPTION
**Pull request by TarasMavrovych**

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+document.getElementById("switchlang").addEventListener("click", function() {
+  const localization = document.getElementById("switchlang");
```
Comment (PavelSukhinin) [inline assets/js/app.js:2]: 1. don't use anonymous functions for event handling
2. get your checkbox inside handler via event

File: assets/css/style.css [hunk 2]
```diff
@@ -29,10 +29,64 @@ body {
...
+  margin: 0 5px;
+}
+
+input[type="checkbox"] {
+  position: relative;
+
+  width: 36px;
+  height: 16px;
+
+  -webkit-appearance: none;
```
Comment (PavloSukhinin) [inline assets/css/style.css]: why with webkit prefix?

File: assets/js/app.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+  localization?.checked ? window.location.href = 'index_en.html': window.location.href = 'index.html'
```
Comment (PavloSukhinin) [inline assets/js/app.js]: 1. don't use anonymous functions for event handling
2. get your checkbox inside handler via event

File: assets/js/app.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+document.getElementById("switchlang").addEventListener("click", function() {
```
Comment (PavloSukhinin) [inline assets/js/app.js]: execute your scripts in strict mode

File: index.html [hunk 5]
```diff
@@ -15,16 +15,22 @@
 </head>
 <body>
   <!-- main section -->
-  <main class="main_section">
+  <main class="main_section" id="ua">
```
Comment (PavloSukhinin) [inline index.html]: I believe you're not using this id

File: assets/js/app.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+"use strict"
+  const localization = document.getElementById("switchlang");
+  function handleClickLang() {
+    localization?.checked ? window.location.href = 'index_en.html': window.location.href = 'index.html'
```
Comment (PavloSukhinin) [inline assets/js/app.js]: make your handler reusable. 
```suggestion
  function handleClickLang(ev) {
   // ev - research this object
   don't use localization?.checked defined in outer scope
```

**Review by PavloSukhinin — CHANGES_REQUESTED** [review]

**Review by PavloSukhinin — APPROVED** [review]

**Pull request merged**